konfera/models/ticket.py

```diff
@@ -58,4 +58,12 @@ def save(self, *args, **kwargs):
             order = Order(price=self.type.price, discount=discount, status=AWAITING, purchase_date=timezone.now())
             order.save()
             self.order = order
-        super(Ticket, self).save(*args, **kwargs)
+            super(Ticket, self).save(*args, **kwargs)
+        else:
+            super(Ticket, self).save(*args, **kwargs)
+            self.order.price = 0
+            self.order.discount = 0
+            for ticket in self.order.ticket_set.all():
```

Comment:
we can have function in order `recalculate_tickets` that will contain this functionality.